modules/sql-engine/src/integrationTest/sql/types/char/test_char.test

```diff
@@ -6,26 +6,24 @@
 query T rowsort
 SELECT ' prefix' || c1 || 'trailing ' FROM (VALUES('1 '::CHAR(5)), (' 2'::CHAR(5)), (''::CHAR(5)), (null)) t(c1);
 ----
- prefix 2   trailing 
- prefix1    trailing 
- prefix     trailing 
+ prefix 2trailing 
+ prefix1 trailing 
```

Comment:
seems we still bit incompatible here with PG, do we need to add right space padding ?

Reply:
We agreed to consider CHAR type as alias to VARCHAR. This means Ignite will never be compliant with PG when dealing with CHAR type.

Current behavior (with missing right padding) closer to what we agreed.